numpy/__init__.py

```diff
@@ -338,11 +338,14 @@ def _mac_os_check():
     import os
     use_hugepage = os.environ.get("NUMPY_MADVISE_HUGEPAGE", None)
     if sys.platform == "linux" and use_hugepage is None:
-        use_hugepage = 1
-        kernel_version = os.uname().release.split(".")[:2]
-        kernel_version = tuple(int(v) for v in kernel_version)
-        if kernel_version < (4, 6):
-            use_hugepage = 0
+        try:
```

Comment:
Maybe it's worth adding a comment here to explain why the try/except is necessary (perhaps referencing the issue)? A little context might help a future reader understand why it was structured this way.

Reply:
thanks for the feedback @rossbar. i have added a comment here.